File: 3-Repeticao/ex9.js
let pares = 0;
let impares = 0;
let soma = 0;

for (let i = 1; i <= 10; i++) {
    let num = parseInt(prompt(`Digite o ${i}º número:`));
    soma += num;
    if (num % 2 === 0) {
        pares++;
    } else {
        impares++;
    }
}

console.log("Pares:", pares);
console.log("Ímpares:", impares);
console.log("Soma total:", soma);

File: 3-/ex9.py
print("==Contagem de impar e par==")
print()

pares = 0
impares = 0
soma = 0

for i in range(1, 11):
    num = int(input(f"Digite o {i}º número: "))
    soma += num
    if num % 2 == 0:
        pares += 1
    else:
        impares += 1

print("Pares:", pares)
print("Ímpares:", impares)
print("Soma total:", soma)
